src/eq.js

```diff
@@ -0,0 +1,157 @@
+  };
+  p5.EQ.prototype = Object.create(Effect.prototype);
+
+  /**
+   * Process an input by connecting it to the EQ
+   * @method  process
+   * @param  {Object} src Audio source
+   */
+  p5.EQ.prototype.process = function (src) {
+    src.connect(this.input);
+  };
+
+  /** 
+   * Set the frequency and gain of each band in the EQ. This method should be
+   * called with 3 or 8 frequency and gain pairs, depending on the size of the EQ. 
+   * ex. eq.set(freq0, gain0, freq1, gain1, freq2, gain2);
+   *
+   * @method  set
+   * @param {Number} [freq0] Frequency value for band with index 0
+   * @param {Number} [gain0] Gain value for band with index 0
+   * @param {Number} [freq1] Frequency value for band with index 1
+   * @param {Number} [gain1] Gain value for band with index 1
+   * @param {Number} [freq2] Frequency value for band with index 2
+   * @param {Number} [gain2] Gain value for band with index 2
+   * @param {Number} [freq3] Frequency value for band with index 3
+   * @param {Number} [gain3] Gain value for band with index 3
+   * @param {Number} [freq4] Frequency value for band with index 4
+   * @param {Number} [gain4] Gain value for band with index 4
+   * @param {Number} [freq5] Frequency value for band with index 5
+   * @param {Number} [gain5] Gain value for band with index 5
+   * @param {Number} [freq6] Frequency value for band with index 6
+   * @param {Number} [gain6] Gain value for band with index 6
+   * @param {Number} [freq7] Frequency value for band with index 7
+   * @param {Number} [gain7] Gain value for band with index 7
+   */
+  p5.EQ.prototype.set = function() {
+    if (arguments.length === this.bands.length * 2) {
+      for (var i = 0; i < arguments.length; i+=2) {
+        this.bands[i/2].freq(arguments[i]);
+        this.bands[i/2].gain(arguments[i+1]);
+      }
+    }
+    else {
+      console.error("Argument mismatch. .set() should be called with " + (this.bands.length*2) +
+        " arguments. (one frequency and gain value pair for each band of the eq)");
+    }
+  };
+
+  /**
+   * Add a new band. Creates a p5.Filter and strips away everything but 
+   * the raw biquad filter. This method returns an abstracted p5.Filter,
+   * which can be added to p5.EQ.bands, in order to create new EQ bands.
+   * @method  _newBand
+   * @private
+   * @param  {Number} freq 
+   * @param  {Number} res  
+   * @return {Obect}      Abstracted Filter
+   */
+  p5.EQ.prototype._newBand = function(freq, res) {
+    var newFilter = new Filter('peaking');
+    newFilter.disconnect();
+    newFilter.set(freq, res);
+    newFilter.biquad.gain.value = 0;
+    delete newFilter.input;
+    delete newFilter.output;
+    delete newFilter._drywet;
+    delete newFilter.wet;
+    return newFilter;
+  }
+
+  p5.EQ.prototype.dispose = function () {
+    Effect.prototype.dispose.apply(this);
+    
+    while (this.bands.length > 0) {
+      delete this.bands.pop().biquad.disconnect();
```

Comment:
nice workaround for the dispose method. I wonder if dispose method could be overwritten (above) to make it useable here in keeping with the Filter API. Maybe that's not necessary as the user wouldn't need to dispose of individual EQ filters...